components/vf-form__checkbox/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 3.0.0
```

Comment:
Question/thought: in general should we make these changes `3.0.0-alpha.1` as they're quite big changes and subject to surprises and further modification?